src/components/Courseware/CoursewareContent/VerticalNav/index.jsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import PropTypes from 'prop-types';
+import { Button } from '@edx/paragon';
+
+
+const VerticalNav = function VerticalNav(props) {
+  return (
+    <div>
+      <Link to={{
```

Comment:
Dug in a little - while this works it is semantically wrong as a `<Link>` becomes an anchor and anchors shouldn't contain buttons...

Will also have accessibility issues as you'd tab to the anchor first, then the button. Instead we can use the withRouter HOC to get history, and then use `history.push` and an onClick for the button...

See: https://stackoverflow.com/questions/45666288/how-to-use-reacts-router-v4-history-push